src/services/utilities.ts

```diff
@@ -1648,14 +1648,25 @@ namespace ts {
      * WARNING: This is an expensive operation and is only intended to be used in refactorings
      * and code fixes (because those are triggered by explicit user actions).
      */
-    export function getSynthesizedDeepClone<T extends Node | undefined>(node: T, includeTrivia = true): T {
-        const clone = node && getSynthesizedDeepCloneWorker(node as NonNullable<T>);
+    export function getSynthesizedDeepClone<T extends Node | undefined>(node: T, includeTrivia = true, renameMap?: Map<[Identifier, number]>, checker?: TypeChecker): T {

```

Comment:
Why `[Identifier, number]`, rather than a type with named members?  Also, I'm assuming the number is used elsewhere - it doesn't appear to be used here.